src/query_parser/v2/lexer.rl

```diff
@@ -136,6 +136,8 @@ main := |*
 
   empty_string => {
     tok.pos = ts-q->raw;
+    tok.s = ts+2;
+    tok.len = te-ts-2;
```

Comment:
Setting `tok.s` past the quotes feels awkward, and the length of an empty string should be 0. Consider
```suggestion
    tok.s = "";
    tok.len = 0;
```

Reply:
It's equivalent, but if this makes you feel more comfortable I'm fine with it